Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
Hi @slaff, the normal applications will use _eagle.app.v6.cpp.ld_ as before. But you are right, this should also include  _eagle.app.v6.cpp.include.ld_ to avoild code duplication.